ocs_ci/ocs/scale_noobaa_lib.py

```diff
@@ -112,3 +114,44 @@ def cleanup(namespace):
     if obc_name_list:
         for i in obc_name_list:
             run_cmd(f"oc delete obc {i} -n {namespace}")
+
+
+def get_endpoint_pod_count(namespace):
+    """
+    Function to query number of endpoint running in the cluster.
+
+    Args:
+        namespace (str): Namespace where endpoint is running
+
+    Returns:
+        endpoint_count (int): Number of endpoint pods
+
+    """
+    endpoint_count = pod.get_pod_count(
+        label=constants.NOOBAA_ENDPOINT_POD_LABEL, namespace=namespace
+    )
+    log.info(f"Number of noobaa-endpoint pod(s) are running: {endpoint_count}")
+    return endpoint_count
+
+
+def get_hpa_utilization(namespace):
+    """
+    Function to get hpa utilization in the cluster.
+
+    Args:
+        namespace (str): Namespace where endpoint is running
+
+    Returns:
+         hpa_cpu_utilization (int): Percentage of CPU utilization on HPA.
+
+    """
+    obj = ocp.OCP()
+    hpa_resources = obj.exec_oc_cmd(
+        command=f"get hpa -n {namespace}", out_yaml_format=False
+    ).split(",")
+    for value in hpa_resources:
+        value = re.findall(r"(\d{1,3})%/(\d{1,3})%", value.strip())
+        value_list = [item for elem in value for item in elem]
+        hpa_cpu_utilization = int(value_list[0])
+        log.info(f"HPA CPU utilization by noobaa-endpoint is {hpa_cpu_utilization}%")
```

Comment:
IMO, the function should simply return the value and the logging needs to be done outside the function. This improves readability.

Reply:
This can be changed. Fixed.